Mailer/README.md

```diff
@@ -876,7 +990,182 @@ Response:
 
 ---
 
-#### GET `/api/v1/mailers/mail-types`
+#### POST `/api/v1/mailers/logs-count-for-email`
+
+Returns number of logs based on given criteria
+
+##### *Headers:*
+
+| Name | Value | Required | Description |
+| --- |---| --- | --- |
+| Authorization | Bearer *String* | yes | API token. |
+
+##### *Body:*
+
+```json5
+{
+  // required 
+  "filter": { // Object/Array; Available filters are delivered_at, clicked_at, opened_at, dropped_at, spam_complained_at, hard_bounced_at
+    "hard_bounced_at": {
+      "from": "2020-04-08", // String; Restrict results to specific from date, optional
```

Comment:
If some specific format is expected in the value, it should be documented here (in the comment)